client/stylesheets/_toCat.scss

```diff
@@ -3788,12 +3793,15 @@ On click the images will be displayed at normal size to complete the effect
     border-radius: 6px;
 }
 
-
+.profileUserNameHidden, .profileUserGraphHidden {
```

Comment:
Selector `profileUserNameHidden` should be written in lowercase with hyphens<br>Selector `profileUserGraphHidden` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line